### PR TITLE
feat: Workspace item

### PR DESCRIPTION
**Pull request by robertkill**
feat: Workspace item

Workspace item

Log: Workspace item
Influence: Workspace item

### CODE REVIEW

**Comment by xzl01** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/linuxdeepin/dde-shell/pull/103#" title="Author self-approved">robertkill</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=linuxdeepin%2Fdde-shell).

<details open>
Needs approval from an approver in each of these files:

- **[debian/deepin/OWNERS](https://github.com/linuxdeepin/dde-shell/blob/master/debian/deepin/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by tsic404 — APPROVED** [review]

**Pull request merged**
